docs/Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    = -W --keep-going -n
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = python -m sphinx
```

Comment:
Presumably no side-effects to this? Just a different way of invoking Sphinx?

Reply:
this is just a more robust way of invoking sphinx-build; not sure when the change happened, but I (as well as CI) was having issues with poetry and sphinx getting along without this. See related issue:

https://github.com/sphinx-doc/sphinx/issues/11218